base/applications/regedit/childwnd.c

```diff
@@ -137,7 +137,7 @@ static void finish_splitbar(HWND hWnd, int x)
     GetClientRect(hWnd, &rt);
     g_pChildWnd->nSplitPos = x;
     ResizeWnd(rt.right, rt.bottom);
-    InvalidateRect(hWnd, &rt, FALSE);
+    InvalidateRect(hWnd, &rt, FALSE); // HACK: See CORE-CORE-19576
```

Comment:
It should be `CORE-19576` and not `CORE-CORE-19576`. No Sure, why I can't use the suggest-feature here atm.

Reply:
Sometimes tie github web-interface has some glitches, indeed.

Reply:
Added suggesto to your comment.

Reply:
Fixed. Thanks.